prime-router/docs/allowlist-gitleaks-false-positives.md

```diff
@@ -0,0 +1,158 @@
+% Allow-listing Gitleaks False Positives
+
+# Introduction
+
+We use [Gitleaks](https://github.com/zricethezav/gitleaks) to scan our repository and commits for sensitive values. The repository is scanned on a schedule-basis through a [GitHub Action](../../.github/workflows/run_gitleaks.yaml) (aka 'nightly run') and your commits are scanned by a [pre-commit hook](getting_started.md#git-hooks) which must be enabled in each of your local repository clones.
+
+Gitleaks uses pattern matching to scan for values that it flags as 'potentially sensitive' if they match the provided pattern(s). This does mean however that it will sometimes report 'hits' that are False Positives. This document describes how to inform gitleaks of a False Positive and suppress warning(s) both for work in progress as well as already committed work.
+
+In all of this, there is one thing to keep in mind:
+
+"**The purpose is not to have non-failing Gitleaks Nightly Runs; the purpose is to not commit sensitive values into our repository and to be able to deal with them ASAP when this does happen, the _side-effect_ thereof is that the Gitleaks Nightly Run succeeds!**"
+
+# Where to configure Gitleaks
+
+Gitleaks is configured through a [TOML](https://en.wikipedia.org/wiki/Toml) file which is located at the following path in your repository: "[.environment/gitleaks/gitleaks-config.toml](../../.environment/gitleaks/gitleaks-config.toml)". This is the configuration file that is used for both the scheduled run and the pre-commit hook.
+
+It is important to understand how the different Gitleaks runs pick up the configuration that applies to each type of run:
+
+* **pre-commit hook**: uses the _configuration file from your branch_, only scans those files that are listed as staged
+* **scheduled/nightly run**: uses the _configuration file as present in the "`master`" branch_ and the rules contained in that version of the file will apply on _all_ commits that it scans in its run. This run spans _all_ commits, in _all_ branches, matching the provided (date-based) filter. In other words: your branch "`foo`" that you pushed up last night, will be scanned by the nightly run using the rules defined in the "`master`" branch! You may have suppressed some false positives in the version of the configuration file in your branch, but unless these are merged into master, they will be ignored by the scheduled run and you will still see failures.
```

Comment:
Bold the last sentence?

Reply:
Yep, fixed in 56093d9b88c30cd2a02bddcd4999ee81571a5947